packages/shared/src/lib/core/wallet/actions/mintNft.ts

```
@@ -13,11 +13,13 @@ import { MintNftParams } from '@iota/wallet'
 import { get } from 'svelte/store'
 import { DEFAULT_TRANSACTION_OPTIONS, OUTPUT_TYPE_NFT } from '../constants'
 import { resetMintNftDetails } from '../stores'
+import { network } from '@core/network'
 
 export async function mintNft(metadata: IIrc27Metadata, quantity: number): Promise<void> {
     try {
         const account = get(selectedAccount)
-        if (!account) {
+        const networkId = get(network)?.getMetadata()?.id
+        if (!account || !networkId) {
             return
```

Do you need to throw an error here like the others?

---

How come this is resolved if the other files all throw an error?